cat-launcher/src/pages/BackupsPage/NewBackupDialog.tsx

```diff
@@ -0,0 +1,96 @@
+import {
+    Dialog,
+    DialogContent,
+    DialogDescription,
+    DialogFooter,
+    DialogHeader,
+    DialogTitle,
+} from "@/components/ui/dialog";
+import { Button } from "@/components/ui/button";
+import {
+    Form,
+    FormControl,
+    FormField,
+    FormItem,
+    FormLabel,
+    FormMessage,
+} from "@/components/ui/form";
+import { Input } from "@/components/ui/input";
+import { useForm } from "react-hook-form";
+import { zodResolver } from "@hookform/resolvers/zod";
+import * as z from "zod";
+import { Textarea } from "@/components/ui/textarea";
+
+const formSchema = z.object({
+    name: z.string().min(1, "Name is required"),
+    notes: z.string().optional(),
+});
+
+interface NewBackupDialogProps {
+    open: boolean;
+    onOpenChange: (open: boolean) => void;
+    onSave: (values: z.infer<typeof formSchema>) => void;
+}
+
+export function NewBackupDialog({
+    open,
+    onOpenChange,
+    onSave,
+}: NewBackupDialogProps) {
+    const form = useForm<z.infer<typeof formSchema>>({
+        resolver: zodResolver(formSchema),
+        defaultValues: {
+            name: "",
```

Comment:
Default value for the name of a manual backup should be {selected_variant}_{timestamp}.